sdk/src/main/java/com/eegeo/mapapi/widgets/RouteViewOptions.java

```diff
@@ -37,6 +38,17 @@ public RouteViewOptions color(int color) {
         return this;
     }
 
+    /**
+     * Sets the forward path color of the RouteView's polylines as a 32-bit ARGB color. The default value is opaque red (0xff000000).
```

Comment:
Hex value for the color looks wrong. We should also update the docs for color attribute as we have changed the default color but the docs still say black.